idris-highlight-input.el

```diff
@@ -84,6 +84,32 @@ See Info node `(elisp)Overlay Properties' to understand how ARGS are used."
                    end-line end-col
                    highlight ))))))
 
+(defun idris-highlight-source-file (hs)
+  (cl-loop
+   for h in hs
+   do (pcase h
+        (`(((:filename ,fn)
+            (:start ,start-line-raw ,start-col-raw)
+            (:end ,end-line-raw ,end-col-raw))
+           ,props)
+         (when (string= (file-name-nondirectory fn)
+                        (file-name-nondirectory (buffer-file-name)))
+           (let ((start-line (if (>=-protocol-version 2 1)
+                                 (1+ start-line-raw)
+                               start-line-raw))
+                 (start-col  (if (>=-protocol-version 2 1)
+                                 (1+ start-col-raw)
+                               start-col-raw))
+                 (end-line   (if (>=-protocol-version 2 1)
+                                 (1+ end-line-raw)
+                               end-line-raw))
+                 (end-col    (if (>= idris-protocol-version 1)
```

Comment:
this is the line that contains fix 
`(>= idris-protocol-version 2 1)` -> ` (>= idris-protocol-version 1)`